distribution/changelog.txt

```diff
@@ -41,6 +41,7 @@
 - Fix: [#11953] Incorrect banner text shade colour on wall text.
 - Fix: [#12062] Inconsistent lift hill sprites on Flying Coaster while inverted (original bug).
 - Fix: [#12068] Incorrect Entrance/Exit location on track design preview. Incorrect track design previews with track that contain diagonal track elements.
+- Fix: [#12093] Staff window tab animation was no longer updating.
```

Comment:
Since this was not present in release 0.2.6, it shouldn't get a changelog entry. I'll fix it.

Reply:
It was as far as i could tell. Only the staff offset was recent

Reply:
OK, re-added it.